kfdefs/base/jupyterhub/notebook-images/ucsls.yaml

```diff
@@ -20,10 +20,10 @@ spec:
   tags:
     - annotations:
         openshift.io/imported-from: quay.io/ucsls/ucsls
-      name: stable
+      name: stable-opf
       from:
         kind: DockerImage
-        name: 'quay.io/ucsls/ucsls:stable'
+        name: 'quay.io/ucsls/ucsls:stable-opf'
```

Comment:
both `stable` and `stable-opf` have the same `SHA` digest, are we sure this rename change is is necessary here?

Reply:
Yes is this not ok?  This so that we have a degree of freedom... We will be updating the image periodically without having to change the image stream yaml.

Reply:
It is fine, it wont cause an issue, if it helps from a naming / semantic perspective it is a good change.

Reply:
@Gregory-Pereira  this is not needed, it upto the user discretion